docker-compose.yml

```diff
@@ -49,12 +49,7 @@ services:
       - db_homs:postgres-homs
       - activiti_homs:activiti
     volumes:
-      - /etc/hydra/homs/homs_configuration.yml:/opt/homs/config/homs_configuration.yml
-      - /etc/hydra/homs/imprint.yml:/opt/homs/config/imprint.yml
-      - /etc/hydra/homs/hbw.yml:/opt/homs/config/hbw.yml
-      - /etc/hydra/homs/database.yml:/opt/homs/config/database.yml
-      - /etc/hydra/homs/activiti.yml:/opt/homs/config/activiti.yml
-      - /etc/hydra/homs/sources.yml:/opt/homs/config/sources.yml
+      - $HOMS:/opt/homs
```

Comment:
Create new docker-compose.dev.yml.
Original docker-compose.yml should remain the same.

Reply:
Created.